components/places/src/places.udl

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+namespace places {};
+
+// Some kind of namespacing for uniffi would be ideal. Multiple udl/macro defns?
+// Everything below is from the crate::storage::history_metadata module...
```

Comment:
> An `interface` here wouldn't work in combination w/ the rest of this component being non-uniffied?

No, I can't quite see how that would work in the current setup :( It would be close though and maybe we can arrange for something, but I'm still thinking we should charge ahead while we work that out.